HyperVExtension/src/HyperVExtension/Models/VirtualMachineCreation/ArchiveExtractionOperation.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace HyperVExtension.Models.VirtualMachineCreation;
+
+/// <summary>
+/// Represents an operation to extract an archive file.
+/// </summary>
+public sealed class ArchiveExtractionOperation : IOperationReport
```

Comment:
"Operation" suffix sounds confusing to me. I'd think this class would execute some operation, but it's only used as a data storage for reporting progress. Perhaps, something like ArchiveExtractionReport, ArchiveExtractionProgressData?

Reply:
Removed thanks